### ISSUE TITLE
Windows slash bugs

### ISSUE THREAD

**Issue by muhibbudins** [romanschejbal/gassetic#65]
I'm run gassetic and few some problem, in osx & linux compile path use "/" slash for directory separator but in windows compile path use "\" backslash. So if i run in browser throw an error "not found" cz 
"\temp\css" will compile to "mpcss". How to fix it?

Thanks for advice :)

Gassetic compiled path :
![1](https://cloud.githubusercontent.com/assets/17744043/22059305/d26e4dbc-dd9e-11e6-857b-6d5e88980cd7.PNG)

Browser compiled path :
![2](https://cloud.githubusercontent.com/assets/17744043/22059306/d27356fe-dd9e-11e6-8bcb-db1a843f32c4.PNG)

**Comment by romanschejbal** [comment]
Oops, that's a bug... Can you please try with the new 1.0.6 version?

**Comment by muhibbudins** [comment]
Hmmm i already use latest version of your library ..

**Comment by muhibbudins** [comment]
But maybe tomorrow i will try upgrade again this library .. Thanks before for advice :D

**Comment by romanschejbal** [comment]
I've released a new version with the bug fix an hour ago..

**Comment by muhibbudins** [comment]
Wow many thanks dude, i will try again tomorrow ..

**Comment by muhibbudins** [comment]
Hello there, i try upgrade it to version 1.0.6 and use `gassetic` command. But script and style tags still use "backslash" like this picture step. Maybe i missed something? Thank you

Update Gassetic :

![update](https://cloud.githubusercontent.com/assets/17744043/22091312/3471711c-de29-11e6-9045-1a904098e614.PNG)

Run `gassetic` command :

![run-command](https://cloud.githubusercontent.com/assets/17744043/22091310/34706eac-de29-11e6-9603-023e995e4d48.PNG)

Code after run command :

![after-command](https://cloud.githubusercontent.com/assets/17744043/22091311/34714322-de29-11e6-804c-662e5f3fdf76.PNG)

My Gassetic config :

```yml
requires:
    freeze: gulp-freeze
    minify: gulp-clean-css
    concat: gulp-concat
    uglify: gulp-uglify
mimetypes:
    # This section contains the formatters for the css files
    css:
        dev:
            outputFolder: web/tmp/css
            webPath:      /tmp/css
            tasks: []
            htmlTag: '<link rel="stylesheet" type="text/css" href="{{ asset("%path%") }}">'
        # In 'prod' mode, use these settings
        prod:
            outputFolder: web/compiled/css  # The output folder for your saving your compiled files
            webPath:      /compiled/css     # The web path for the compiled files
            # Run these tasks on your compiled files
            tasks:
                - { name: minify }
                - { name: concat, args: '%filename%' }
                # This is a cache busting gulp plugin that appends
                #  an md5 of the contents to the filename
                - { name: freeze }
            htmlTag: '<link rel="stylesheet" type="text/css" href="{{ asset("%path%") }}">' # custom html tag

        # This is the list of source files to apply the above settings
        files:
            backend.css: # This is the output filename
                - assets/legacy/adminLTE/css/datepicker/bootstrap-datepicker.css
                - assets/legacy/adminLTE/css/AdminLTE.css
                - bower_components/toastr-fa/toastr.css
                - assets/css/backend.css
            home.css:
                - assets/legacy/adminLTE/css/morris/morris.css
                - assets/legacy/adminLTE/css/jvectormap/jquery-jvectormap-1.2.2.css
                - assets/legacy/adminLTE/css/datepicker/datepicker3.css
                - assets/legacy/adminLTE/css/daterangepicker/daterangepicker-bs3.css
                - assets/legacy/adminLTE/css/bootstrap-wysihtml5/bootstrap3-wysihtml5.min.css
            promo-scheduler.css:
                - assets/legacy/adminLTE/css/daterangepicker/daterangepicker-bs3.css
            order-edit.css:
                - assets/legacy/adminLTE/css/datepicker/datepicker3.css
                - assets/legacy/adminLTE/css/daterangepicker/daterangepicker-bs3.css
            report-shoppingcart-review.css:
                - assets/legacy/adminLTE/css/morris/morris.css
            admin-dashboard.css: 
                - assets/css/style_dashboard.css

        # Watch these files for changes (optional)
#        watch:
#            - assets/**/*.less
#            - assets/**/*.css

    # This section contains the formatters for your JS files
    js:
        dev:
            outputFolder: web/tmp/js
            webPath:      /tmp/js
            tasks: []
            htmlTag: '<script src="{{ asset("%path%") }}"></script>'
        prod:
            outputFolder: web/compiled/js  # Save the files here
            webPath:      /compiled/js     # Specify the web path
            tasks:
                - { name: concat, args: '%filename%' }
                - { name: uglify, args: { mangle: false } }
            htmlTag: '<script src="{{ asset("%path%") }}"></script>'

        # Here is a list of files to apply the above tasks to
        files:
            ckeditor-style.js:
                - assets/js/ckeditor-style.js
            backend.js: # This is the output filename
                - bower_components/bootstrap/dist/js/bootstrap.min.js
                - assets/legacy/adminLTE/js/jquery-ui.min.js
                - bower_components/toastr-fa/toastr.js
                - assets/legacy/adminLTE/js/raphael-min.js
                - assets/legacy/adminLTE/js/daterangepicker/daterangepicker.js
                - assets/legacy/adminLTE/js/datepicker/bootstrap-datepicker.js
                - bower_components/sidr/dist/jquery.sidr.min.js
                - assets/legacy/adminLTE/js/app.js
                - assets/legacy/adminLTE/js/demo.js
                - assets/js/general.js
                - assets/js/backend.js
                - assets/js/global-search.js
            home.js: 
                - assets/legacy/adminLTE/js/morris/morris.min.js
                - assets/legacy/adminLTE/js/sparkline/jquery.sparkline.min.js
                - assets/legacy/adminLTE/js/jvectormap/jquery-jvectormap-1.2.2.min.js
                - assets/legacy/adminLTE/js/jvectormap/jquery-jvectormap-world-mill-en.js
                - assets/legacy/adminLTE/js/jqueryKnob/jquery.knob.js
                - assets/legacy/adminLTE/js/bootstrap-wysihtml5/bootstrap3-wysihtml5.all.min.js
                - assets/legacy/adminLTE/js/dashboard.js
            auction.js: 
                - assets/js/auction/auction.js
            groupbuy.js: 
                - assets/js/groupbuy/groupbuy.js
            payment.js: 
                - assets/js/payment/payment.js
            product.js: 
                - assets/js/product/default.js
            pointreward.js: 
                - assets/js/pointreward/pointreward.js
            order-edit.js: 
                - assets/js/transaction/order-edit.js
            supplier-order.js: 
                - assets/js/transaction/supplier-order.js
            seo-config.js: 
                - assets/js/seo/config.js
            seo-keyword.js: 
                - assets/js/seo/keyword.js
            seo-meta.js: 
                - assets/js/seo/meta.js
            report-shoppingcart-review.js:
                - assets/legacy/adminLTE/js/morris/morris.min.js

    fonts:
        prod:
            outputFolder: web/compiled/css/fonts
            tasks: []
            autoRenaming: false

        dev:
            outputFolder: web/tmp/css/fonts
            tasks: []
            autoRenaming: false

        files:
            desktop:
                - assets/fonts/*

    images:
        prod:
            outputFolder: web/compiled/css/img
            tasks: []
            autoRenaming: false

        dev:
            outputFolder: web/tmp/css/img
            tasks: []
            autoRenaming: false

        files:
            desktop:
                - assets/img/desktop/**/*

# This is the list of files/paths to search and find the replacement tags to insert the
# generated <script> or stylesheet tags
replacementPaths:
    - app/Resources/views/*.html.twig
    - src/**/*.html.twig

default:
    - js
    - css
    - images
    - fonts
```

**Comment by romanschejbal** [comment]
That's strange, may I ask what's your npm version? I've released a new version 1.0.7 that should replace all backslashes for the webPath. Can you try now?

**Comment by muhibbudins** [comment]
Nice it's work, i try upgrade to version 1.0.7. Thank you dude
